platform/.openshift/keycloak.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+    apiVersion: template.openshift.io/v1
```

Comment:
It will be more benefitial to use operator for us as we could create realm from Operator CR etc.

Reply:
Sure. 

I only needed this for my local testing. If we have keycloak, kafka services running then that's awesome.

You can still use the `yarn keycloak:init` command to initialise keycloak. All that's needed is the url and admin user credentials.

Reply:
Yep. We can wrap that into cli and use some assumption `dataservices` namespace containing all URLs etc.